.circleci/config.yml

```diff
@@ -53,6 +53,7 @@ jobs:
     - run: go test -v -tags api
     - run: go test -v -tags end2end
     - run: go test -v -tags trap
+    - run: go test -timeout 60s -v -tags marshal -fuzz '^FuzzUnmarshal$'
```

Comment:
FYI, you probably want `-fuzztime` instead of `-timeout` here.

from `go help testflag`

```
	-fuzztime t
	    Run enough iterations of the fuzz target during fuzzing to take t,
	    specified as a time.Duration (for example, -fuzztime 1h30s).
		The default is to run forever.
	    The special syntax Nx means to run the fuzz target N times
	    (for example, -fuzztime 1000x).
```